sdk/js/src/utils/array.ts

```diff
@@ -88,15 +88,15 @@ export const tryUint8ArrayToNative = (
       }
       return humanAddress("terra", a.slice(-20));
     }
+  } else if (chainId === CHAIN_ID_INJECTIVE) {
+    return humanAddress("inj", a.slice(-20));
```

Comment:
https://github.com/certusone/wormhole/pull/1396/files#diff-a3eb09e88d85dc2b2581978b2811d00c168f0b0a57c18f094e62cc79e75de3f7R13 Does this handle `0xpeggy` prefixed addresses?

Reply:
Removed peggy addresses.